src/main/Welcome.java

```diff
@@ -0,0 +1,549 @@
+		welcomePanel.add(helpButton);
+
+		JLabel usernameLabel = new JLabel("Username:");
+		usernameLabel.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		usernameLabel.setBounds(616, 13, 80, 25);
+		welcomePanel.add(usernameLabel);
+
+		JLabel passwordLabel = new JLabel("Password:");
+		passwordLabel.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		passwordLabel.setBounds(876, 13, 80, 25);
+		welcomePanel.add(passwordLabel);
+
+		usernameField = new JTextField();
+		usernameField.setColumns(10);
+		usernameField.setBounds(706, 13, 160, 25);
+		welcomePanel.add(usernameField);
+
+		passwordField = new JPasswordField();
+		passwordField.setBounds(966, 13, 160, 25);
+		welcomePanel.add(passwordField);
+
+		JButton signInButton = new JButton("Sign In");
+		signInButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				welcomePanel.setVisible(false);
+				dashboardPanel.setVisible(true);
+			}
+		});
+		signInButton.setBounds(796, 49, 160, 25);
+		welcomePanel.add(signInButton);
+
+		JButton createAccountButton = new JButton("Create an account");
+		createAccountButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				welcomePanel.setVisible(false);
+				dashboardPanel.setVisible(true);
+			}
+		});
+		createAccountButton.setBounds(966, 49, 160, 25);
+		welcomePanel.add(createAccountButton);
+
+		JLabel directionsLabel = new JLabel("Please sign in or create an account to continue");
+		directionsLabel.setFont(new Font("Tahoma", Font.PLAIN, 24));
+		directionsLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		directionsLabel.setBounds(0, 380, 1136, 73);
+		welcomePanel.add(directionsLabel);
+
+		JLabel logoLabel = new JLabel("Logo");
+		logoLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		logoLabel.setBounds(0, 20, 211, 35);
+		taskbarPanel.add(logoLabel);
+
+		JButton accountButton = new JButton("My Account");
+		accountButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				introPanel.setVisible(false);
+				accountPanel.setVisible(true);
+			}
+		});
+		accountButton.setBounds(25, 180, 160, 23);
+		taskbarPanel.add(accountButton);
+
+		JButton quizButton = new JButton("Take Quiz");
+		quizButton.setBounds(25, 240, 160, 23);
+		taskbarPanel.add(quizButton);
+
+		JButton resultsButton = new JButton("View Results");
+		resultsButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				accountPanel.setVisible(false);
+				resultsPanel.setVisible(true);
+			}
+		});
+		resultsButton.setBounds(25, 300, 160, 23);
+		taskbarPanel.add(resultsButton);
+
+		JButton helpButton2 = new JButton("Help");
+		helpButton2.setBounds(45, 500, 120, 23);
+		taskbarPanel.add(helpButton2);
+
+		JButton signOutButton = new JButton("Sign Out");
+		signOutButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				dashboardPanel.setVisible(false);
```

Comment:
It would also be helpful to separate the panels into classes (i.e. separate files) so when the other groupmates integrate their feature into the screen, it would make merging everything a lot easier (and reduce the number of pesky merge conflicts)